docs/www/kubernetes-connectivity.md

```diff
@@ -139,8 +141,12 @@ Let’s enable external connectivity for all supported APIs:
      - port: 8082
      - external:
          enabled: true
+    schemaRegistry:
+      port: 8081
+      external:
+        enabled: true
     adminApi:
-    - port: 9644
+     - port: 9644
```

Comment:
NIT: I don't have x-ray vision in my eyes, but the alignment seems odd. The yaml list can be align with later `a` or later `m`. Don't know why its align with latter `m`.